src/vs/workbench/common/theme.ts

```diff
@@ -144,6 +144,12 @@ export const TAB_UNFOCUSED_INACTIVE_FOREGROUND = registerColor('tab.unfocusedIna
 	hc: Color.white
 }, nls.localize('tabUnfocusedInactiveForeground', "Inactive tab foreground color in an unfocused group. Tabs are the containers for editors in the editor area. Multiple tabs can be opened in one editor group. There can be multiple editor groups."));
 
+export const DASHBOARD_TAB_ACTIVE_BACKGROUND = registerColor('tab.dashboardActiveBackground', {
+	dark: '#e1f0fe',
```

Comment:
waiting to hear back from Raden on what the color here should be for dark theme

Reply:
😆

Reply:
@kisantia how about copy the style of portal?

Reply:
updated with values from Raden